internal/distro/rhel86/distro.go

```diff
@@ -610,7 +632,8 @@ func newDistro(distroName string) distro.Distro {
 
 	// Shared Services
 	edgeServices := []string{
-		"NetworkManager.service", "firewalld.service", "sshd.service",
+		// TODO(runcom): move fdo-client-linuxapp.service to presets?
```

Comment:
we have a follow up card for this so we can get this in and I'll clean up once that's done (this allows us to test the flow as having a preset)